City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-7651: Daycare search without ontology word details

# [UHF-7651](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7651)

## What was done
* Removed daycare search's dependency to `tpr_ontology_word_details` migration.
* The search now uses unit's categories field for the daycare options.

## How to install
* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-7651-daycare-search-without-owd`
  * `make fresh`
* Run `make drush-cr`

## How to test
* Run `make shell` and then
  * `drush migrate:import tpr_unit --update`
  * This will add daycare related categories for relevant units.
* Go to some page that includes the daycare search.
* Using the Options dropdown, make sure the daycare search still works.
* Also check translations and daycare search's other functionality.

## Designers review
